fe/src/components/Header/Header.tsx

```diff
@@ -43,6 +47,15 @@ export default function Header(): JSX.Element {
             ))}
           </HStack>
         </HStack>
+        <HStack spacing={12} ml="auto" title={colorMode === 'light' ? 'Switch to dark theme' : 'Switch to light theme'}>
```

Comment:
Also as you can see, this is  exact copy-paste of the component code from `ColorMode.tsx`, why copy existing code, if we could already use it here. You could import the new component in this file and then use it here as one line:
```
<ColorMode />
```